src/wmtk/Mesh.hpp

```diff
@@ -288,16 +288,24 @@ class Mesh : public std::enable_shared_from_this<Mesh>
      */
     virtual bool is_ccw(const Tuple& tuple) const = 0;
     /**
-     * @brief check if all tuple simplices besides the cell are on the boundary
+     * @brief check if a simplex of codimension 1 is a boundary simplex
      *
      * @param tuple
      * @return true if all tuple simplices besides the cell are on the boundary
      * @return false otherwise
      */
-    virtual bool is_boundary(const Tuple& tuple) const = 0;
+    bool is_boundary(const Tuple& codim_1_simplex) const;
```

Comment:
I feel like this version of `is_boundary` should not exist in the API. There is quite a risk of people using it unintentionally for the wrong simplices.